rustls/src/msgs/handshake.rs

```diff
@@ -890,12 +890,20 @@ impl Codec for ClientHelloPayload {
 }
 
 impl ClientHelloPayload {
+    pub fn get_extensions(&self) -> &[ClientExtension] {
+        if let Some(ref exts) = self.extensions {
```

Comment:
I'd write this as `self.extensions.as_deref().unwrap_or(&[])`.

(And `get_()` prefixes are discouraged in the API naming guidelines, which should still work since methods and fields don't share namespaces.)